gatsby/config-options/gatsby-plugin-feed-options.tsx

```diff
@@ -2,19 +2,18 @@ import {
   documentToHtmlString,
   Options,
 } from '@contentful/rich-text-html-renderer';
-import {
-  BASE_URL,
-  DEFAULT_META_IMAGE_URL_PATH,
-  RSS_FEED_URL,
-} from './constants';
-import * as ReactDOMServer from 'react-dom/server';
-import React from 'react';
 import { BLOCKS } from '@contentful/rich-text-types';
+import * as ReactDOMServer from 'react-dom/server';
+import { Converter } from 'showdown';
 import {
   ContentfulCustomModel,
   EmbeddedAssetType,
 } from '../../src/types/contentful.types';
-import { Converter } from 'showdown';
+import {
+  BASE_URL,
+  DEFAULT_META_IMAGE_URL_PATH,
+  RSS_FEED_URL,
+} from './constants';
 
```

Comment:
This part changed due to "optimizing imports" which runs once the file is changed.
It was not intended to do this, but it doesn't hurt so I will not revert that. 🤠